client/components/account-balances/balance-block.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { formatCurrency } from 'wcpay/utils/currency';
+import Loadable from 'components/loadable';
+
+/**
+ * BalanceBlockProps
+ *
+ * @typedef {Object} BalanceBlockProps
+ *
+ * @property {string} title        Title of the balance block. eg "Available Funds".
+ * @property {string} currencyCode Currency code of the balance block.
+ * @property {number} [amount]     Optional. The balance amount.
+ * @property {boolean} [isLoading] Optional. Whether the balance block is loading.
+ */
+interface BalanceBlockProps {
```

Comment:
🤔 `type` vs `interface`. I prefer `type` as a default but there is not a consistent approach within WCPay at this stage.

Reply:
TIL. Down the rabbit hole I got on `type` vs `interface` 😄